src/shared/components/post/post-listing.tsx

```diff
@@ -207,9 +207,9 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
           <>
             {this.listing()}
             {this.state.imageExpanded && !this.props.hideImage && this.img}
-            {this.showBody &&
-              unescapeHTML(post.url) &&
-              unescapeHTML(post.embed_title) && <MetadataCard post={post} />}
+            {this.showBody && post.url && post.embed_title && (
```

Comment:
Asf or the reason for the failure on L290, its because you removed a check for `unescape(post.embed_title)`

Reply:
Can you help me understand what is causing the error here so I can prevent it in the future? Looking at the [file history - 2fa7c11](https://github.com/LemmyNet/lemmy-ui/blob/2fa7c11666455d28c395d7b8b830faf1c94b2158/src/shared/components/post/post-listing.tsx#L209), it has `{this.showBody && post.url && post.embed_title &&` for that line, instead of separated out to ` unescapeHTML` for each. 

I think what you are saying is that on L209 for the `post.`'s to _not have the `unescapeHTML`_, but instead only  needed on L290. Is that correct?

Reply:
The error is telling you that unescapeHtml needs a `string`, not a `string | undefined`. So the easiest way to fix it, is in L283, do add `&& post.embed_title` to the check.